main/ui/src/main/java/org/cryptomator/ui/fxapp/FxApplicationModule.java

```diff
@@ -34,17 +36,17 @@
 	}
 
 	@Provides
-	@Named("windowIcon")
+	@Named("windowIcons")
 	@FxApplicationScoped
-	static Optional<Image> provideWindowIcon() {
+	static List<Image> provideWindowIcons() {
 		if (SystemUtils.IS_OS_MAC) {
-			return Optional.empty();
-		}
-		try (InputStream in = FxApplicationModule.class.getResourceAsStream("/window_icon_32.png")) { // TODO: use some higher res depending on display?
-			return Optional.of(new Image(in));
-		} catch (IOException e) {
-			return Optional.empty();
+			return new ArrayList<>();
```

Comment:
I prefer the static utility methods over constructors (gives the JVM more room for internal optimizations and future updates): 

```suggestion
			return Collections.emptyList();
```

Reply:
Fixed.